### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by laxmankhanal**

### CODE REVIEW

File: Vaccination/ViewController/CustomVaccineFormTVC.swift [hunk 1]
```diff
@@ -33,9 +33,14 @@ class CustomVaccineFormTVC: UITableViewController, UITextFieldDelegate {
         self.ageLabel.text = dateToString(child.dateOfBirth!)
     }
 
-    @IBAction func cancelVaccination(sender: AnyObject) {
+    @IBAction func cancelVaccination(sender: UIButton) {
+        let alert = UIAlertController(title: "Discard information and exit?", message: "You won't be able to get same information in form again.", preferredStyle: UIAlertControllerStyle.Alert)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/CustomVaccineFormTVC.swift]: Too long message

File: Vaccination/ViewController/MainVC.swift [hunk 2]
```diff
@@ -373,7 +376,9 @@ extension MainVC: VaccinationCellDelegate {
 
     func cellForVaccinationDate(cell: VaccinationCell) {
         self.keyBoardRequestor = KeyBoardRequester.CustomDateButton
-        showDatePickerWithButtons(cell)
+        if !isHistorySegment {
```
Comment (sanjaymhj) [inline Vaccination/ViewController/MainVC.swift]: use segmentSelected.isUpcoming()
and delete isHistorySegment

File: Vaccination/ViewController/MainVC.swift [hunk 3]
```diff
@@ -19,6 +19,7 @@ class MainVC: UIViewController, KCFloatingActionButtonDelegate {
     var firstView = UIView()
     var startButton = UIButton()
     var fab = KCFloatingActionButton()
+    var isHistorySegment = false
```
Comment (sanjaymhj) [inline Vaccination/ViewController/MainVC.swift]: no need of this flag.... 
segmentSelected.isUpcoming() does the same work

File: Vaccination/ViewController/CustomVaccineFormTVC.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/CustomVaccineFormTVC.swift]: :ok:

File: Vaccination/ViewController/MainVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/MainVC.swift]: :ok:

File: Vaccination/ViewController/MainVC.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/MainVC.swift]: :ok:

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 4]
```diff
@@ -91,8 +91,8 @@ class AddChildFormTVC: UITableViewController, UITextFieldDelegate {
         presentViewController(actionSheet, animated: true, completion: {})
     }
 
-    @IBAction func cancelAddingChild(sender: UIBarButtonItem) {
-        let alert = UIAlertController(title: "Alert", message: "Discard information and exit", preferredStyle: UIAlertControllerStyle.Alert)
+    @IBAction func cancelAddingChild(sender: UIButton) {
+        let alert = UIAlertController(title: "Discard information and exit?", message: "", preferredStyle: UIAlertControllerStyle.Alert)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: Discard and exit?

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 5]
```diff
@@ -121,7 +121,7 @@ class AddChildFormTVC: UITableViewController, UITextFieldDelegate {
             error!.removeAll()
         } else {
             if editChild {
-                let alert = UIAlertController(title: "Alert", message: "Are you sure to edit information", preferredStyle: UIAlertControllerStyle.Alert)
+                let alert = UIAlertController(title: "Are you sure to edit information?", message: "", preferredStyle: UIAlertControllerStyle.Alert)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: Confirm?

**Comment by sanjaymhj** [comment]
:+1:

**Pull request merged**